server.go

```diff
@@ -1814,13 +1829,13 @@ func (s *Server) executeShowSeriesStatement(stmt *influxql.ShowSeriesStatement,
 		// Loop through series IDs getting matching tag sets.
 		for _, id := range ids {
 			if s, ok := m.seriesByID[id]; ok {
-				values := make([]string, 0, len(r.Columns))
+				values := make([]interface{}, 0, len(r.Columns))
```

Comment:
@otoolep  this is an important part of the fix because it fixes the final result format.